core/src/com/unciv/logic/automation/civilization/ReligionAutomation.kt

```diff
@@ -31,13 +31,8 @@ object ReligionAutomation {
             return
         }
         
-        if (civInfo.religionManager.remainingFoundableReligions() == 0 && civInfo.religionManager.religionState == ReligionState.Pantheon) {
+        if (civInfo.religionManager.remainingFoundableReligions() == 0 ) {
             buyGreatPerson(civInfo)
-            return
-        }
-
-        // We don't have a religion and no more change of getting it :(
-        if (civInfo.religionManager.religionState <= ReligionState.Pantheon) {
             tryBuyAnyReligiousBuilding(civInfo)
```

Comment:
So we will only try buying a religious building on the turn when we're buying a great prophet to found our religion?! And even that, before we actually found that religion because we haven't actually activated the great prophet yet? This sounds much worse actually!